src/devtools/client/debugger/src/components/SecondaryPanes/NewScopes.tsx

```diff
@@ -17,15 +17,25 @@ import { getPreferredGeneratedSources } from "ui/reducers/sources";
 import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
 import { pickScopes } from "ui/suspense/scopeCache";
 
-import { getSelectedFrameId } from "../../selectors";
+import { getSeekLock, getSelectedFrameId } from "../../selectors";
 import { ConvertedScope, convertScopes } from "../../utils/pause/scopes/convertScopes";
 import styles from "./NewObjectInspector.module.css";
 
+function LoadingInfo() {
+  return <div className="pane-info">Loading…</div>;
+}
+
 function ScopesRenderer() {
   const replayClient = useContext(ReplayClientContext);
   const sourcesById = sourcesByIdCache.read(replayClient);
   const preferredGeneratedSources = useAppSelector(getPreferredGeneratedSources);
   const selectedFrameId = useAppSelector(getSelectedFrameId);
+  const seekLock = useAppSelector(getSeekLock);
+
+  if (seekLock) {
+    return <LoadingInfo />;
+  }
```

Comment:
conceptually it could make sense to have a similar check in the Frames but:
- I couldn't make those 2 to desynchronize right now
- I'm not quite sure where exactly I would squeeze it there (things are rendered recursively there)
- it's not worth putting more effort into this rn ;p